tests/monit_test/test_monit_service_status.py

```diff
@@ -0,0 +1,26 @@
+"""
+Test the running status of Monit service
+"""
+import logging
+
+import pytest
+
+logger = logging.getLogger(__name__)
+
+pytestmark = [
+    pytest.mark.topology('any')
+]
+
+
+def test_monit_service_status(duthost):
+    """
+    @summary: Test the running status of Monit service by analyzing the command
+              output of "sudo systemctl status monit.service | grep Active".
+    """
+    monit_service_status_info = duthost.shell("sudo systemctl status monit.service | grep Active")
```

Comment:
should just do sudo monit status

and then check the return value, whether it is zero or not.

Reply:
True. Checking the return value of `sudo monit status` is zero will ensure that not only is Monit running, but that it is configured correctly.

Reply:
Great suggestion! Fixed.